aplib.net-demo/Assets/Scripts/Player/UpdateCameraPos.cs

```diff
@@ -2,9 +2,9 @@
 
 public class UpdateCameraPos : MonoBehaviour
 {
-    [SerializeField] Transform cameraTransform;
+    [SerializeField] private Transform cameraTransform;
```

Comment:
```suggestion
    [SerializeField] private Transform _cameraTransform;
```

Reply:
Yes I just made some changes style-wise for now, still need to fix those points